hack/lib/serverless.bash

```diff
@@ -204,6 +204,82 @@ EOF
   logger.success 'Knative Kafka has been installed sucessfully.'
 }
 
+function ensure_kafka_no_auth {
+  logger.info 'Ensure Knative Kafka using no Kafka auth'
+
+  # Apply Knative Kafka
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: ${EVENTING_NAMESPACE}
+spec:
+  source:
+    enabled: true
+  channel:
+    enabled: true
+    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
+EOF
+
+  # shellcheck disable=SC2016
+  timeout 900 '[[ $(oc get knativekafkas.operator.serverless.openshift.io knative-kafka -n $EVENTING_NAMESPACE -o=jsonpath="{.status.conditions[?(@.type==\"Ready\")].status}") != True ]]'  || return 7
+
+  logger.success 'Knative Kafka has been set to use no auth sucessfully.'
+}
+
+function ensure_kafka_tls_auth {
+  logger.info 'Ensure Knative Kafka using TLS auth'
+
+  # Apply Knative Kafka
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: ${EVENTING_NAMESPACE}
+spec:
+  source:
+    enabled: true
+  channel:
+    enabled: true
+    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9093
+    authSecretNamespace: default
+    authSecretName: my-tls-secret
+EOF
+
+  # shellcheck disable=SC2016
+  timeout 900 '[[ $(oc get knativekafkas.operator.serverless.openshift.io knative-kafka -n $EVENTING_NAMESPACE -o=jsonpath="{.status.conditions[?(@.type==\"Ready\")].status}") != True ]]'  || return 7
```

Comment:
Should we replace these with

```
oc wait -n $EVENTING_NAMESPACE knativekafkas.operator.serverless.openshift.io knative-kafka --for=condition=Ready --timeout 900 
```

?

@cardil any issue with that?

Reply:
Oc wait doesn't give any feedback as I remember

Reply:
Feedback in which sense? This command would only print `False != True` anyway and eventually fail. I guess no info would be lost by using `oc wait`

Reply:
I meant you just wait, nothing appear on screen.